Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionTanFragment.kt

```diff
@@ -56,7 +56,7 @@ class SubmissionTanFragment : Fragment() {
             is BadRequestException -> DialogHelper.DialogInstance(
                 requireActivity(),
                 R.string.submission_error_dialog_web_test_paired_title,
-                R.string.submission_error_dialog_web_test_paired_body,
+                R.string.submission_error_dialog_web_test_paired_body_tan,
```

Comment:
there are already related Strings..
If you want, you can use them:
 is BadRequestException -> DialogHelper.DialogInstance(
                requireActivity(),
                R.string.submission_error_dialog_web_tan_invalid_title,
                R.string.submission_error_dialog_web_tan_invalid_body,
                R.string.submission_error_dialog_web_tan_invalid_button_positive,
                null,
                true,
                ::goBack
            )

Kind Regards.
Bernd

Reply:
Hm... but this key constant has not the value as in https://jira.itc.sap.com/browse/EXPOSUREAPP-2552.

Reply:
submission_error_dialog_web_tan_invalid_body = Die erzeugte Übermittlungs-TAN ist ungültig. Bitte kontaktieren Sie die technische Hotline über App-Informationen → Technische Hotline.
but in the screenshot in the Ticket it says otherwise.

Reply:
Luka just updated the ticket (it was confusing)...